word_guess.rb

```diff
@@ -0,0 +1,159 @@
+  all_guesses << guess
+  new_game.correct
+  elsif guess == letter_array[5]
+  unguessed_answer_letters_array[5] = letter_array[5]
+  all_guesses << guess
+  new_game.correct
+  elsif guess == letter_array[6]
+  unguessed_answer_letters_array[6] = letter_array[6]
+  all_guesses << guess
+  new_game.correct
+  elsif guess != letter_array[0] && guess != letter_array[1] && guess != letter_array[2] && guess != letter_array[3] && guess != letter_array[4] && guess != letter_array[5] && guess != letter_array[6]
+  count = count -1
+  all_guesses << guess
+    if count == 4
+    puts "\nOh no! A bunny is coming to your yard!"
+      require 'colorize'
+      bunny_one =   "\n========".colorize(:cyan)
+      bunny_two =   "\n ()_()  "
+      bunny_thr =   "\n(=o.o=) "
+      bunny_for =   "\n(')_(')*"
+      bunny_fiv =   "\n========".colorize(:cyan)
+      puts bunny_one + bunny_two + bunny_thr + bunny_for + bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    elsif count == 3
+    puts "\nOh no! Another bunny is coming to your yard!"
+      require 'colorize'
+      second_bunny_one = "\n=====================".colorize(:cyan)
+      second_bunny_two = "\n   ()__()     ()_()  "
+      second_bunny_thr = "\n  (=o.o=)     (^+^)  "
+      second_bunny_for = "\n  (w)_(w)*  *(')_(') "
+      second_bunny_fiv = "\n=====================".colorize(:cyan)
+      puts second_bunny_one + second_bunny_two + second_bunny_thr + second_bunny_for + second_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    elsif count == 2
+    puts "\nOh no! Another bunny is coming to your yard!"
+      require 'colorize'
+      third_bunny_one = "\n===============================".colorize(:light_blue)
+      third_bunny_two = "\n  (')_(')     ()_()     ()_()  "
+      third_bunny_thr = "\n  ('o+o')     (^+^)     (O.o)  "
+      third_bunny_for = "\n  (@)_(@)*  *(')_(')  *(')(')  "
+      third_bunny_fiv = "\n===============================".colorize(:light_blue)
+      puts third_bunny_one + third_bunny_two + third_bunny_thr + third_bunny_for + third_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    elsif count == 1
+      puts "\nOh no! Another bunny is coming to your yard!"
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+      require 'colorize'
+      fourth_bunny_one = "\n========================================".colorize(:cyan)
+      fourth_bunny_two = "\n  (')_(')     ()_()     ()_()   ()_()   "
+      fourth_bunny_thr = "\n  ('o+o')     (^+^)     (O.o)   (-.-)   "
+      fourth_bunny_for = "\n  (@)_(@)*  *(')_(')  *(')(')  (')(')*  "
+      fourth_bunny_fiv = "\n========================================".colorize(:cyan)
+      puts fourth_bunny_one + fourth_bunny_two + fourth_bunny_thr + fourth_bunny_for + fourth_bunny_fiv
+      puts "You have #{count} incorrect guesses before bunnies take over!"
+    end
+  end
+
+
+  #WIN
```

Comment:
The fact that you have these comments here mean to me that it would be great to put this code inside of a method